src/components/Modal.jsx

```diff
@@ -195,6 +248,8 @@ Modal.defaultProps = {
   title: 'Please give me a title :)',
   content: 'Please give me some content :)',
   image: '',
+  sound: '',
```

Comment:
Ah, nope, I'm wrong. You can't default a string to a boolean, it does do typing on defaults. I coulda sworn that wasn't true! Oh well. Sorry about that!